orchestra/static/orchestra/timing/timer/directives.js

```diff
@@ -0,0 +1,105 @@
+(function() {
+  'use strict';
+
+  angular
+    .module('orchestra.timing')
+    .directive('workTimer', function() {
+      return {
+        restrict: 'E',
+        scope: {
+          taskId: '=taskId',
+        },
+        bindToController: true,
+        controllerAs: 'workTimer',
+        templateUrl: '/static/orchestra/timing/timer/timer.html',
+        controller: function($http, $location, $scope, $timeout, timeEntries) {
+          var workTimer = this;
+
+          workTimer.popoverTemplate = '/static/orchestra/timing/timer/popover.html';
```

Comment:
wrap in `$static` - @joshblum static file processing works when the template url is set to a variable right?